docs/src/pages/index.tsx

```diff
@@ -7,41 +7,74 @@ function HomepageHeader() {
 
   return (
     <header>
-      <section className="text-center m-6 p-12 border border-red-400 rounded-[50px] bg-slate-200 dark:bg-immich-dark-gray">
+      <div className="left-0 top-[calc(45%)] w-screen h-screen absolute -z-10">
         <img
-          src={isDarkTheme ? 'img/immich-logo-stacked-dark.svg' : 'img/immich-logo-stacked-light.svg'}
-          className="md:h-60 h-44 mb-2 antialiased rounded-none"
+          src={isDarkTheme ? 'img/logomark-dark.png' : 'img/logomark-light.png'}
+          className="h-full w-full mb-2 antialiased -z-10"
           alt="Immich logo"
         />
-        <div className="sm:text-2xl text-lg md:text-4xl mb-12 sm:leading-tight">
-          <p className="mb-1 font-medium text-immich-primary dark:text-immich-dark-primary">
-            Self-hosted photo and <span className="block"></span>
-            video management solution<span className="block"></span>
+        <div className="w-screen h-screen absolute left-0 top-0 backdrop-blur-3xl bg-immich-bg/40 dark:bg-transparent"></div>
+      </div>
+      <section className="text-center pt-12 sm:pt-24 bg-immich-bg/50 dark:bg-immich-dark-bg/80">
+        <img
+          src={isDarkTheme ? 'img/logomark-dark.png' : 'img/logomark-light.png'}
+          className="h-[115px] w-[115px] mb-2 antialiased rounded-none"
+          alt="Immich logo"
+        />
+        <div className="mt-8">
+          <p className="text-3xl md:text-5xl sm:leading-tight mb-1 font-extrabold text-black/90 dark:text-white px-4">
+            Self-hosted{' '}
+            <span className="text-immich-primary dark:text-immich-dark-primary">
+              photo and <span className="block"></span>
+              video management{' '}
+            </span>
+            solution<span className="block"></span>
+          </p>
+
+          <p className="max-w-1/4 m-auto mt-4 px-4">
+            Easily back up, organize, and manage your photos on your own server. Immich helps you
+            <span className="sm:block"></span> browse, search and organize your photos and videos with ease, without
+            sacrifice your privacy.
           </p>
```

Comment:
```suggestion
          <p className="max-w-1/4 m-auto mt-4 px-4">
            Easily back up, organize, and manage your photos on your own server. Immich helps you
            <span className="sm:block"></span> browse, search and organize your photos and videos with ease, without
            sacrificing your privacy.
          </p>
```

Maybe there should be a line break before "Immich" here.

Reply:
It is intentional for this purpose 
<img width="921" alt="image" src="https://github.com/user-attachments/assets/d4df316e-7247-42a4-8197-65b373720063">